README.md

```diff
@@ -1,9 +1,24 @@
+
 # Operators for Qiskit
 
-###  - [Qiskit developer's exp operator](https://github.com/husky-parul/glowing-quantum/tree/qiskit-dev-op/operators-examples/qiskit-dev-operator)
+## Overwiew
+
+These operators integrates your quantum workloads on OpenShift. It provides a development environment to implement quantum algorithms and runs them either on IBM quantum computers or simulators.
 
-Operator sets up the development environment with an integrated Jupyter Notebook and pre installed dependecy for running quantum circuits on [IBMQ Account](https://quantum-computing.ibm.com/) using [Qiskit](https://qiskit.org/)
+## Getting started
+
+1. Qiskit developer's experience operator
     
-Example notebook implementing Grover's Search Algorithm
+    Operator sets up a development environment with an integrated [Jupyter Notebook](https://hub.gke.mybinder.org/user/ipython-ipython-in-depth-azjvu4ak/notebooks/examples/Notebook/Notebook%20Basics.ipynb) and pre-installed dependencies for running quantum circuits on [IBMQ Account](https://quantum-computing.ibm.com/) using [Qiskit](https://qiskit.org/).
+
+    Example notebook implementing [Grover's Search Algorithm](https://qiskit.org/textbook/ch-algorithms/grover.html)
+
+    For getting started guides, installation, deployment and test Qiskit developer's experience operator follow [here](https://github.com/husky-parul/glowing-quantum/tree/qiskit-dev-op/operators-examples/qiskit-dev-operator).
+
+## Contributing
+
+Operators for Qiskit  is a community driven project and we welcome contributions. See Contributing to get started.
+
+## Report a Bug
 
-###  - Qiskit runtime operator
\ No newline at end of file
+For filing bugs, suggesting improvements, or requesting new features, please open an  [issue](https://github.com/yard-turkey/glowing-quantum/issues)
```

Comment:
I think this repo needs to move to the ET repo.

Reply:
are we leaving it in yard-turkey for this pr?

Reply:
I believe that Ryan merged this PR into ET repo so all follow on work and future PRs should be handled there! So this PR probably should be closed @husky-parul @jeffvance agree?